test/evergreen.yml

```diff
@@ -1783,6 +1792,31 @@ tasks:
             set -o verbose
 
             ./time_shift_test.sh /usr/local/lib/faketime/libfaketimeMT.so.1 0-1 2>&1
+  
+  - name: format-stress-sanitizer-test
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags: CC="clang -fsanitize=address" CFLAGS="-g -fno-omit-frame-pointer" --enable-diagnostic --with-builtins=lz4,snappy,zlib
+      - command: shell.exec
+        params: 
+          working_dir: "wiredtiger/build_posix/test/format"
+          script: |
+            set -o errexit
+            set -o verbose
+
+            export ASAN_OPTIONS="detect_leaks=1:abort_on_error=1:disable_coredump=0"
+            if [ -f /usr/bin/llvm-symbolizer ]; then
+              ASAN_SYMBOLIZER_PATH="/usr/bin/llvm-symbolizer"
+            else
+              ASAN_SYMBOLIZER_PATH=`ls /usr/bin/llvm-symbolizer* | tail -1`
+            fi
+            export ASAN_SYMBOLIZER_PATH
+      - func: "format test script"
+        vars:
+          # run for 24 hours ( 24 * 60 = 1440 minutes), stop at first error, use default config
+          format_test_script_args: "-F -t 2"
```

Comment:
Ignore the 2 min argument. This is to quickly test the PR. Final commit will have this set as 1440.

Reply:
Even for a shorter-interval test, 2 mins is probably too short, because the default timer setting in `test/format/CONFIG.stress` is 4 mins, which means each format job will run ~4 mins (could be more depending on how quick the format test process responds to the stop signal). From your patch build you can see all 4 format jobs got killed prematurely due to the 2 mins format.sh `-t` setting.

Reply:
Suggest to create a patch build to run the 24hr-testing sooner than later as the test takes time to finish :)

Reply:
Good point. I am addressing the comments and will start a evergreen build hopefully before EOD.